src/setup/devnet/index.js

```diff
@@ -561,8 +561,17 @@ export class Devnet {
                     ];
 
                     // create heimdall folder
-                    shell.exec(`sudo mkdir -p /var/lib/heimdall`)
-                    shell.exec(`sudo chmod 777 -R /var/lib/heimdall/`)
+                    for (let i = 0; i < this.totalNodes; i++) {
+                        await execa('ssh', [
+                            `-o`, `StrictHostKeyChecking=no`, `-o`, `UserKnownHostsFile=/dev/null`,
+                            `-i`, `~/cert.pem`,
+                            `${this.config.devnetBorUsers[i]}@${this.config.devnetBorHosts[i]}`,
+                            `sudo mkdir -p /var/lib/heimdall && sudo chmod 777 -R /var/lib/heimdall/`
+                        ], {stdio: getRemoteStdio()})
+                        
+                    }
+                    /*shell.exec(`sudo mkdir -p /var/lib/heimdall`)
```

Comment:
can we remove the comments if not needed?

Reply:
Yep taken care here: [7e27cdf](https://github.com/maticnetwork/matic-cli/pull/91/commits/7e27cdf2e41b2a3f04dc3eef1217c4a73af982f7)